src/commands/datasets/create.ts

```diff
@@ -7,7 +7,7 @@ import { error, success } from '../../lib/outputs.js';
 import { getLoggedClientOrThrow } from '../../lib/utils.js';
 
 export class DatasetsCreateCommand extends ApifyCommand<typeof DatasetsCreateCommand> {
-	static override description = 'Creates a new Dataset on your account';
+	static override description = 'Creates named dataset for storing structured data on your account.';
```

Comment:
the dataset name is optional, if its missing, this is not creating a named dataset, right?